RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,867 @@
+          the responses to ????duration after the begin_query_time. 
+        required: false
+        style: simple
+        explode: false
+        $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Time'
+      responses:
+        "200":
+          description: List of future capacity details was retrieved successfully. 
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/QueryFutureCapacityResponse'
+        "400":
+          description: One or more input parameters were missing or invalid.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "401":
+          description: "Bearer access token was not provided in Authorization header, token could not be decoded, or token was invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "403":
+          description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "404":
+          description: The requested Entity could not be found.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "429":
+          description: The client issued too many requests in a short period of time.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+      security:
+      - Authority:
+        - utm.strategic_coordination
+    put:
+      tags:
+      - Resource capacities
+      - p2p_uam 
+      summary: Submit a new or changed temporary resource capacity to a peer. 
+      operationId: updateFutureResourceCapacities
+      parameters:
+      - name: entityid
+        in: path
+        description: EntityID of the operational intent.
+        required: true
+        style: simple
+        explode: false
+        schema:
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/EntityID'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PutFutureCapacitiesRequest'
+        required: true
+      responses:
+        "201":
+          description: Future temporary capacities were successfully received.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorReport'
```

Comment:
This shouldn't be an error report, right, since it the 200 response?

Reply:
Yes, I think we have this in a couple places throughout the specs.  We must have a successful acknowledgement schema... need to find and replace